docs/csharp/language-reference/keywords/interpolated-strings.md

```diff
@@ -47,7 +50,7 @@ If the interpolated string contains other characters with special meaning in an
 
 [!code-cs[interpolated-strings4](../../../../samples/snippets/csharp/language-reference/keywords/interpolated-strings4.cs#1)]  
 
-Verbatim interpolated strings use the `@` character in addition to using the `$` character for interpolated strings. You can learn more about verbatim strings in the [string](strings.md) topic. The following code is a modified version of the previous snippet using a verbatim interpolated string:
+Verbatim interpolated strings use the `@` character followed by the `$` character. For more information about verbatim strings, see the         [string](strings.md) topic. The following code is a modified version of the previous snippet using a verbatim interpolated string:
```

Comment:
This change makes the text incorrect. The order of `@` and `$` needs to be reversed.

Reply:
It would probably be better something like "as for normal strings, prefixing a verbatim string (`@`, yada, yada, yada) with `$` will be interpreted as an interpolation".

Conceptually, it's not a `$` in front of the `@` but a `$` in front of a string literal in any of its allowed forms.